Source/Fluxor/DependencyInjection/FluxorOptions.cs

```diff
@@ -53,6 +56,35 @@ public FluxorOptions(IServiceCollection services)
 			return this;
 		}
 
+		/// <summary>
+		/// The Store Lifetime that should be used when registering Fluxor features/reducers/effects/middleware
+		/// </summary>
+		/// <param name="lifecycle">the lifecycle to use</param>
+		/// <returns>Options</returns>
+		/// <remarks>
+		/// <list type="bullet">
+		/// <item>
+		/// <term>LifecycleEnum.Scoped</term>
+		/// <description>(default) Create a new instance for each new request</description>
+		/// </item>
+		/// <item>
+		/// <term>LifecycleEnum.Singleton</term>
+		/// <description>Create a new instance on first request and reuse for rest of application lifetime</description>
+		/// <para>
+		/// NOTE: indicating Singleton should be done only for exceptional cases.
+		/// For example, in MAUI/Blazor hybrid applications, the main MAUI application is a different scope then each BlazorWebView component
+		/// and state needs to be shared across all scopes of the application
+		/// </para>
+		/// <para>
+		/// This value should only be set once during the configuration of Fluxor
+		/// </para>
+		/// </remarks>
+		public FluxorOptions WithServiceLifetime(StoreLifetime lifecycle)
```

Comment:
This is all great!
As the param type is `StoreLifetime` could we have the name `WithLifetime`. My fault again, I'm sorry, but I've only just spotted it.

Reply:
no problem - seeing things in context and changing/improving is 'a good thing'